MoltenVK/MoltenVK/GPUObjects/MVKDeviceMemory.h

```diff
@@ -138,6 +133,8 @@ class MVKDeviceMemory : public MVKVulkanAPIDeviceObject {
 	/** Returns the Metal resource options used by this memory allocation. */
 	inline MTLResourceOptions getMTLResourceOptions() { return mvkMTLResourceOptions(_mtlStorageMode, _mtlCPUCacheMode); }
 
+	/** Returns the Metal texture underlying this memory allocation. */
+	inline id<MTLTexture> getMTLTexture() { return _mtlTexture; }
```

Comment:
We're not using `inline` anymore in this context, since it's almost certainly ignored by the compiler, and just letting the compiler be the boss.

Reply:
I'll remove this `inline` and all of the above then to keep consistency!